src/libraries/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -401,6 +405,7 @@ public sealed partial class FileSystem
         public static void MkDir(string Path) { }
         public static void Print(int FileNumber, params object[] Output) { }
         public static void PrintLine(int FileNumber, params object[] Output) { }
+        [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")]
```

Comment:
Rename is windows specific? That seems odd.

Reply:
I looked at the implementation, it does throw PNSE on non-windows. Weird.

Reply:
Welcome to `My` 😁